components/core/src/clp_s/OutputHandlerImpl.cpp

```
@@ -24,6 +25,28 @@ using std::string;
 
 namespace clp_s {
+namespace {
+/**
+ * Connects to the results cache and returns the requested collection.
+ * @tparam OperationFailedT The type of exception to throw on failure.
+ * @param uri
+ * @param collection
+ * @param client Returns the connected client.
+ * @return The collection.
+ */
+template <typename OperationFailedT>
```

Cppcheck (2.21.0)

[style] The function 'get_filename' is never used.

(unusedFunction)